packages/design/src/idemeum/Card.tsx

```diff
@@ -3,10 +3,11 @@ import { Text, Box, Flex } from 'design';
 import { useEffect, useState } from 'react';
 import { CircleCheck } from 'design/Icon';
 import '../assets/idemeum/fonts/fonts.css'
+import cfg from 'teleport/config';
 
 export default function Card({ message, redirectUrl, showIcon }: { message?: string, redirectUrl?: boolean, showIcon?: boolean }) {
 
-    let loginUrl = "";
+    let loginUrl = cfg.routes.login;
```

Comment:
We are initializing the loginUrl conditionally

    if(redirectUrl) {
            var hostName = new URL(window.location.href).hostname;
            loginUrl = hostName.replace(".remote.", ".");
        }
Can you please double check why its not reflected